packages/core/src/browser/shell/view-contribution.ts

```diff
@@ -142,6 +142,7 @@ export abstract class AbstractViewContribution<T extends Widget> implements Comm
         }
         this.quickView?.registerItem({
             label: this.viewLabel,
+            location: this.options.viewContainerId ?? this.options.defaultWidgetOptions.area ?? "",
```

Comment:
The following is causing the build to fail (singlequotes):

```suggestion
            location: this.options.viewContainerId ?? this.options.defaultWidgetOptions.area ?? '',
```